internal/jimm/access.go

```diff
@@ -804,3 +804,62 @@ func (j *JIMM) ListGroups(ctx context.Context, user *openfga.User, pagination pa
 	}
 	return groups, nil
 }
+
+// OpenFGACleanup queries OpenFGA for all existing tuples, tries to resolve each tuple and removes those
+// that JIMM cannot resolved - orphaned tuples. JIMM not being able to resolve a tuple means that the
+// corresponding entity has been removed from JIMM's database.
+//
+// This approach to cleaning up tuples is intended to be temporary while we implement
+// a better approach to eventual consistency of JIMM's database objects and OpenFGA tuples.
+func (j *JIMM) OpenFGACleanup(ctx context.Context) error {
+	var (
+		continuationToken string
+		err               error
+		tuples            []ofga.Tuple
+	)
+	for {
+		tuples, continuationToken, err = j.OpenFGAClient.ReadRelatedObjects(ctx, openfga.Tuple{}, 20, continuationToken)
+		if err != nil {
+			zapctx.Error(ctx, "reading all tuples", zap.Error(err))
+			return err
+		}
+		zapctx.Debug(ctx, "removing orphaned tuples", zap.Any("tuples", tuples))
```

Comment:
This log line is logging **all** tuples, not the ones that are orphaned.

Reply:
done